libs/components/pages/src/lib/modules/link-list/link-list.component.html

```diff
@@ -28,7 +28,16 @@ <h2 class="sky-font-heading-4 sky-margin-stacked-sm">
           'sky-margin-stacked-sm': 'modern'
         }"
       >
-        <a *ngIf="link.permalink.url" [skyHref]="link.permalink.url">
+        <a
+          *ngIf="link.permalink.url && hasProtocol(link.permalink.url)"
```

Comment:
Since this is in an `ngFor` I understand need to make an exception to the "don't call functions in templates" rule, but it should use a pure pipe instead of a function call. That way it will only be evaluated when the input value changes instead of every change detection cycle.

Reply:
Switched to pipes.